src/main.rs

```diff
@@ -43,29 +39,30 @@ async fn main() -> Result<(), LemmyError> {
   env_logger::init();
   let settings = Settings::init().expect("Couldn't initialize settings.");
 
-  // Set up the r2d2 connection pool
+  // Set up the deadpool connection pool
```

Comment:
Most of this PR is boilerplate and can be ignored, this is the only main change, setting up the other pool with a tokio runtime.